src/htm/regions/SPRegion.cpp

```diff
@@ -290,41 +289,11 @@ Spec *SPRegion::createSpec() {
           "inhibition logic will insure that at most N columns remain ON "
           "within a local inhibition area, where N = localAreaDensity * "
           "(total number of columns in inhibition area). "
-          "Mutually exclusive with numActiveColumnsPerInhArea. "
-          " Default ``-1.0`` which means disabled.",
+	  "Default 0.1 (10%)",
           NTA_BasicType_Real32,             // type
           1,                                // elementCount
           "",                               // constraints
-          "-1.0",                           // defaultValue
-          ParameterSpec::ReadWriteAccess)); // access
-
-  ns->parameters.add(
-      "numActiveColumnsPerInhArea",
```

Comment:
Is there any way you could keep the network API piece intact?

Maybe you could repurpose the deleted code from updateBoostFactorsGlobal_ to make the SP's setter getter methods keep working, and just convert to/from the internal format (local area density)

Reply:
Nevermind, I don't really see how to keep the parameter in the network API and not in the underlying SP, at least not easily.